### PR TITLE
Check warnings

### PR DESCRIPTION
**Pull request by todi1856**
* Fix warnings in AndroidLogcatBuildInfo.cs https://github.com/Unity-Technologies/com.unity.mobile.logcat/issues/73
* Added test, it works by trying to test a project which has csc.rsp file which enables warnings to be as errors in C# script.
* Added one more fix with regex https://github.com/Unity-Technologies/com.unity.mobile.logcat/pull/74/commits/31fbdb28e85d005c35129f33a154526d2fdf7d7c

Note: we didn't add csc.rsp in SampleProject1, because that project reference a performance test package which has warnings inside !

### CODE REVIEW

File: format.ps1 [hunk 1]
```diff
@@ -1,2 +1,2 @@
-perl "${Home}/unity-meta/Tools/Format/format.pl" com.unity.mobile.android-logcat/Editor com.unity.mobile.android-logcat/Tests/Editor TestProjects/SampleProject1/Assets TestProjects/ForceCrash/Assets
+perl "${Home}/unity-meta/Tools/Format/format.pl" com.unity.mobile.android-logcat/Editor com.unity.mobile.android-logcat/Tests/Editor TestProjects/SampleProject1/Assets TestProjects/ForceCrash/Assets TestProjects/TestWarnings/Assets
```
Comment (vincent-unity) [inline format.ps1]: You need to update the `format.yml` as well.

File: com.unity.mobile.android-logcat/Editor/AndroidLogcatBuildInfo.cs [hunk 2]
```diff
@@ -1,3 +1,4 @@
+#if PLATFORM_ANDROID
```
Comment (vincent-unity) [inline com.unity.mobile.android-logcat/Editor/AndroidLogcatBuildInfo.cs]: Just noticed we're all using `PLATFORM_ANDROID` in this package, probably should use `UNITY_ANDROID` instead.

File: format.ps1 (hunk 1 above)
Comment (todi1856) [reply format.ps1]: What exactly should I update? format.yml has this line

    - perl unity-meta/Tools/Format/format.pl --dry-run com.unity.mobile.android-logcat/Editor com.unity.mobile.android-logcat/Tests/Editor TestProjects

that means it takes all project by default

Comment (vincent-unity) [reply format.ps1]: Oh, they're not same. Not a problem now.

**Pull request merged**